### PR TITLE
Developer

### PR DESCRIPTION
**Pull request by pedromtcosta**

### CODE REVIEW

File: MovieApp/MovieApp/MovieApp/ViewModels/MovieListPageViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            _movieService = movieService;
+            _dialogService = dialogService;
+            Title = "Browse Movies";
+
+            LoadInitialData();
+        }
+
+        public async void LoadInitialData()
+        {
+            IsBusy = true;
```
Comment (ghost) [inline MovieApp/MovieApp/MovieApp/ViewModels/MovieListPageViewModel.cs]: It is a good practice to catch any exception inside an async void method. Also if there is any exception within the method, the IsBusy flag will not be set to false.

File: MovieApp/MovieApp/MovieApp/App.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+namespace MovieApp
+{
+    public partial class App : PrismApplication
+    {
+
+		public App ()
+		{
+			InitializeComponent();
+
+            MainPage = new MainPage();
```
Comment (ghost) [inline MovieApp/MovieApp/MovieApp/App.xaml.cs]: Wrong identation, space mixed with tabs

File: MovieApp/MovieApp/MovieApp/ViewModels/MovieListPageViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+            }
+            else
+            {
+                await _dialogService.DisplayAlertAsync("Error", result.Error, "OK");
+            }
+        }
+
+        private async Task GetMovies()
+        {
+            IsBusy = true;
```
Comment (ghost) [inline MovieApp/MovieApp/MovieApp/ViewModels/MovieListPageViewModel.cs]: same here, the flag won't be set to false if there is any exception

File: MovieApp/MovieApp/MovieApp/ViewModels/BaseViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using Xamarin.Forms;
+
+namespace MovieApp.ViewModels
+{
+    public class BaseViewModel : BindableBase
+    {
+        bool isBusy = false;
```
Comment (ghost) [inline MovieApp/MovieApp/MovieApp/ViewModels/BaseViewModel.cs]: This should be a private property right?
Please follow your code standards: private bool _isBusy

**Review by ghost — COMMENTED** [review]
Very good prism solution. Unit tests were excellent with a good code coverage. Well done!

**Comment by pedromtcosta** [comment]
Just updated the code to fit with the observations made from the code review

**Comment by pedromtcosta** [comment]
Not relevant anymore

**Pull request closed without merging**